wireapp/wire-ios-mono
# fix: Unable to import backup - WPB-9372

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-9372" title="WPB-9372" target="_blank"><img alt="Bug" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10803?size=medium" />WPB-9372</a>  [iOS] Beta unable to import backup from Production
  </td></table>
  <br />
 
























### Issue
If a user who already has 7 active clients wants to restore a backup created from an app version with an old database version, the app will stuck on the landing screen.

### Solution
After migration, create a new `AuthenticationCoordinator` and go to the `.createCredentials` authentication step. Since we are creating a new authentication coordinator, we no longer have the password from the first step and need to prompt for a password to remove one of the clients. 

**Note:** we didn't ask for a password to delete clients during login, but it wasn't consistent with other platforms. QA and product agree to ask a password each time.

### Testing
1. Create a backup with the previous version of the database.
2. You should have 7 clients for the user you are going to use.
3. Import backup.

**Expected:**
- There should be a screen "Too many devices".
- After deleting one of the clients, the login process should be completed.

### Checklist
